Makefile

```diff
@@ -7,11 +7,11 @@ DESTDIR=/usr/local
 # Used to populate version variable in main package.
 VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
```

Comment:
nit: maybe because `m` is new to me, `dirty` seems to be a more usual notifier of this state. I can get used to `m` but seems less automatically clear what it means.